src/noRecordFound.js

```diff
@@ -31,6 +30,93 @@ function executeOverride() {
   logEvent("[DOL_DABI] Viewed SSN Not Found page");
 }
 
+function updateLogo() {
+  const logo = document.getElementsByTagName("img")[0];
```

Comment:
Dust: maybe add a check in case logo doesn't exist

Reply:
Good call!